sdcm/sct_events/base.py

```diff
@@ -109,7 +119,7 @@ def __init__(self, severity: Severity = Severity.UNKNOWN):
         self.severity = severity
         self._ready_to_publish = True
         self.event_id = str(uuid.uuid4())
-        self.log_level = logging.INFO
+        self.log_level = LOG_LEVEL_MAPPING.get(severity, logging.ERROR)
```

Comment:
Why ERROR?

Reply:
this is the default, if we'll have severity with isn't in the `LOG_LEVEL_MAPPING` mapping